content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+filters for the :guilabel:`Appraisal Analysis` report.
+
+To view the available filters, click the :icon:`fa-caret-down` :guilabel:`(caret down)` icon in the
+:guilabel:`Search...` bar.
+
+.. image:: reporting/filter-appraisals.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+In the :guilabel:`Filters` section on the left-side of the resulting drop-down menu, three options
+are presented:
+
+- :guilabel:`To Start`: select this filter to **only** present appraisals that have been scheduled
+  (a date populates the :guilabel:`Appraisal Date` field on the appraisal form), but have **not**
+  been confirmed. These appear in gray.
+- :guilabel:`Appraisal Sent`: select this filter to **only** present appraisals that have been
+  scheduled and confirmed, but have **not** been completed. This includes appraisals that have been
+  confirmed but not started, as well as appraisals that are in-process. These appraisals appear in
+  orange.
+- :guilabel:`Done`: select this filter to **only** present appraisals that have been completed and
+  marked as :guilabel:`Done`. These appraisals appear in yellow.
+
+It is possible to select as many or as few filters as desired.
+
+Custom filters
+**************
+
+Appraisals that have been canceled appear in red on the :guilabel:`Appraisal Analysis` report, but
+there is no preconfigured filter to show only cancelled appraisals.
+
+To view only cancelled appraisals, click the :icon:`fa-caret-down` :guilabel:`(caret down)` icon in
+the :guilabel:`Search...` bar.
+
+Next, click :guilabel:`Add Custom Filter` in the :guilabel:`Filters` section, and a :guilabel:`Add
+Custom Filter` pop up window loads.
+
+Using the drop-down menu, select :guilabel:`Status` for the :guilabel:`ID` field, then select
+:guilabel:`Cancelled` for the second drop-down field. Click the :guilabel:`Add` button, and only
+appraisals that have been cancelled appear.
+
+.. image:: reporting/custom-filter.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set to only show cancelled appraisals.
+
+Group appraisals
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Appraisal Analysis` report is grouped by :guilabel:`Department`, by default. This
+grouping can be removed, and/or different groupings can be added.
+
+To view the available grouping options, click the :icon:`fa-caret-down` :guilabel:`(caret down)`
+icon in the :guilabel:`Search...` bar.
+
+Active groupings have a green :icon:`fa-check` :guilabel:`(checkmark)` next to them. Click on a
+grouping option to change its current status, either from active to inactive (or vice versa). Any
+configuration of groupings can be selected.
+
+The available default :guilabel:`Group By` options are:
+
+- :guilabel:`Employee`: groups all appraisals by employee.
+- :guilabel:`Department`: groups all appraisals by department.
+- :guilabel:`Status`: groups all appraisals by their status.
+
```

Comment:
Same reasoning as the Filters section, this can be covered simply by linking to the Odoo essentials documentation on grouping.

Also, move the custom anchor to this heading :)
```suggestion
.. _appraisals/skills-report:
```